lib/state.ml

```diff
@@ -1,5 +1,7 @@
 open Board
+open Player
 
 type game_state = {
   board : board;
+  player_list : player list;
```

Comment:
this field name could be more concise. i suggest `players`, what do you think?

Reply:
I think players is better, will make this change.